rfcs/models/infrastructure/relays.md

```diff
@@ -73,11 +72,32 @@ Once a driver is designed for a given DLT, it can be reused in any network built
 
 ## Relay Functional API
 
-_TBD: List API function specs exposed to application clients, relays, and drivers respectively. Message formats will be described in 'formats/communication'_
+### API for Application Client
+
+- **RequestState(networks.networks.NetworkQuery): returns common.ack.Ack**
+    This endpoint is for the client application to request remote state via local relay. Since this request is asynchronous w.r.t the network, the request info/state machine is stored in a database on the requesting relay indexed via `request_id`. This `request_id` can be used to poll for status and the response using `GetState` endpoint. The local relay then sends the Query to the remote relay via an API available for relay-relay communication (explained below). This API takes a message of type [NetworkQuery](../../formats/communication/relay.md#networkquery) as argument, and returns [Ack](../../formats/views/request.md#ack) message.
```

Comment:
- Say "a client application (i.e., above the contracts) running in a DLT network".
- Why is Query capitalized? I don't believe the `Query` message is used here, is it? That structure is not used by the relay, as far as I know.
- Say "returns an ACK" (capitalize)